test/package.json

```diff
@@ -0,0 +1,7 @@
+{
+  "dependencies": {
```

Comment:
To prevent unexpected behavior modern moog-require won't import something as an apostrophe module unless it is a project level npm dependency. For tests this means adding a package.json as a placeholder, it is not actually consulted by npm.